server/controllers/establishmentRollController.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, RequestHandler, Response } from 'express'
+import EstablishmentRollService from '../services/establishmentRollService'
+
+export default class EstablishmentRollController {
+  constructor(private readonly establishmentRollService: EstablishmentRollService) {}
+
+  public getEstablishmentRoll(): RequestHandler {
+    return async (_req: Request, res: Response) => {
+      const { clientToken, user } = res.locals
```

Comment:
Isn't the clientToken stored in req.middleware now?

Reply:
Yep I did do it to this service: https://github.com/ministryofjustice/hmpps-digital-prison-services/pull/118

Reply:
Good point.. perhaps I was developing off an old version and it is broken since rebase. Will check